File: book_kit/src/main/java/net/lzbook/kit/data/bean/CacheTaskConfig.java
package net.lzbook.kit.data.bean;

import com.google.gson.annotations.SerializedName;
import java.util.List;

public class CacheTaskConfig {
    @SerializedName("bookAuthor")
    public String bookAuthor;
    @SerializedName("bookName")
    public String bookName;
    @SerializedName("bookSourceId")
    public BookSourceIdCacheTaskConfig bookSourceId;
    @SerializedName("fileUrlList")
    public List<String> fileUrlList;
    @SerializedName("lastChapterId")
    public BookSourceIdCacheTaskConfig lastChapterId;
    @SerializedName("version")
    public int version;

    public static class BookSourceIdCacheTaskConfig {
        @SerializedName("counter")
        public int counter;
        @SerializedName("date")
        public long date;
        @SerializedName("machineIdentifier")
        public int machineIdentifier;
        @SerializedName("processIdentifier")
        public int processIdentifier;
        @SerializedName("time")
        public long time;
        @SerializedName("timeSecond")
        public int timeSecond;
        @SerializedName("timestamp")
        public int timestamp;

        public String toString() {
            return "BookSourceIdCacheTaskConfig{timestamp=" + this.timestamp + ", machineIdentifier=" + this.machineIdentifier + ", processIdentifier=" + this.processIdentifier + ", counter=" + this.counter + ", timeSecond=" + this.timeSecond + ", time=" + this.time + ", date=" + this.date + '}';
        }
    }

    public String toString() {
        return "DataCacheTaskConfig{bookSourceId=" + this.bookSourceId + ", bookName='" + this.bookName + '\'' + ", bookAuthor='" + this.bookAuthor + '\'' + ", lastChapterId=" + this.lastChapterId + ", version=" + this.version + ", fileUrlList=" + this.fileUrlList + '}';
    }
}

File: basic/src/main/java/com/ding/basic/bean/CacheTaskConfig.kt
package com.ding.basic.bean

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class CacheTaskConfig: Serializable {

    companion object {
        val USE_CHAPTER_BY_CHAPTER = 60001
    }

    @SerializedName("bookAuthor")
    var bookAuthor: String? = null
    @SerializedName("bookName")
    var bookName: String? = null
    @SerializedName("bookSourceId")
    var bookSourceId: String? = null
    @SerializedName("fileUrlList")
    var fileUrlList: List<String>? = null
    @SerializedName("lastChapterId")
    var lastChapterId: String? = null
    @SerializedName("version")
    var version: Int = 0

    override fun toString(): String {
        return "DataCacheTaskConfig{bookSourceId=" + this.bookSourceId + ", bookName='" + this.bookName + '\'' + ", bookAuthor='" + this.bookAuthor + '\'' + ", lastChapterId=" + this.lastChapterId + ", version=" + this.version + ", fileUrlList=" + this.fileUrlList + '}'
    }
}
